### ISSUE TITLE
Should highlight element consistently

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#171]
**Expected behavior**
The element should be highlighted consistently

**Actual behavior**
Sometimes space near by the element is getting highlighted instead of the element itself
![highlight_near_element](https://user-images.githubusercontent.com/6310197/45799008-a48db980-bcc9-11e8-89a8-70317b49aa0a.png)

**Steps to replicate**
> You may have to run the following script multiple times. The highlight varies. However the highlight is near the required element.
* Script
```
const { openBrowser, goto, text, click, below } = require('taiko');
const assert = require("assert");

(async () => {
    try {
        await openBrowser({headless:false});
        await goto("https://react-shopping-cart-67954.firebaseapp.com/");
        await click(text("M"));
        await click("Add to cart",below("Dark Thug Blue-Navy T-Shirt"));
        await screenshot()
        assert.equal((await text("CHECKOUT").exists()).description,'Exists')
    } catch (e) {
        console.error(e);
    } finally {
        closeBrowser();
    }
})();
```
**Version**
```
commit - eb776a3e0aac915c63c0a8a77c6de8ffc2ccfed8
```
Blocks #163

**Comment by NivedhaSenthil** [comment]
This is happening because the DOM takes time to load even after the XHR is complete. Since 'Frame Load' event is also not triggered there is no other particular event that we could wait by default. 
The filter action actually triggers 'childNodeRemoved' as it is particular for this case user can explicitly wait for it to make sure filtering is done before the next action. 
`await client().DOM.childNodeRemoved();`

**Comment by NivedhaSenthil** [comment]
May be taiko should take an option like `{waitForNodeRemoved:true}` on click to wait for it. 
This should not block #163 as highlight is done correctly where click is happening.

**Comment by NivedhaSenthil** [comment]
Taiko waits for `firstMeaningfulPaint` whenever there is a `firstPaint` happening, this scenario is covered by that. Fix available in 435a9ce
